apps/backend/src/modules/contributors/contributor.controller.ts

```diff
@@ -167,22 +192,23 @@ class ContributorController extends BaseController {
 	 */
 	private async findAll(
 		options: APIHandlerOptions<{
-			query: { projectId?: string } & PaginationQueryParameters;
+			query: ContributorGetAllQueryParameters;
 		}>,
 	): Promise<APIHandlerResponse> {
-		if (options.query.projectId) {
-			const projectId = Number(options.query.projectId);
+		const { contributorName, hasHidden, orderBy, page, pageSize, projectId } =
+			options.query;
 
-			return {
-				payload: await this.contributorService.findAllByProjectId({
-					projectId,
-				}),
-				status: HTTPCode.OK,
-			};
-		}
+		const query = {
+			...(contributorName ? { contributorName } : {}),
+			...(hasHidden ? { hasHidden: (hasHidden as unknown) === "true" } : {}),
```

Comment:
It should be a string, not unknown. Validation schema should only allow 'true' or 'false'

Reply:
And probably we need a mapper or helper to  parse boolean value

Reply:
I tried `as string`, but typescript doesn't like it, as he think that `hasHidden` has boolean type, and I'm getting lint error that I can't convert boolean to string

Reply:
![image](https://github.com/user-attachments/assets/60b6f143-76c4-4cfc-87ad-f4a2b822d67f)

Reply:
Ok, let's leave it with unknown then